component/DashBoard/component/DashboardGoalsAvatar.jsx

```diff
@@ -0,0 +1,77 @@
+import { Dropdown, Tooltip } from "antd";
+import React from "react";
+import { getFirstLetter } from "../../../helpers/truncateString";
+
+const avatarCount = 1;
+const remainingAvatarCount = avatarCount - 1;
+
+function DashboardGoalsAvatar({ activeGoalUsers }) {
+  return (
+    <div className="flex items-center">
+      {Number(activeGoalUsers.length) > 0 &&
+        activeGoalUsers.map((data, index) =>
+          index < avatarCount ? (
+            <Tooltip
+              title={data?.assignee?.first_name}
+              placement="top"
+              key={index + "users"}
+            >
+              <div
+                className={`${
+                  index === 0
+                    ? "bg-cyan-500 "
+                    : index === 1
+                    ? "bg-orange-600 -ml-2"
+                    : "bg-green-600 -ml-2"
+                } 
+            z-${index === 0 ? "0" : `${index}0`}
```

Comment:
explain the condition

Reply:
first item will have z-index 0 second will have z-index 10 and so on
its tailwind classes